File: lib/apollo-stub.ts
/**
 * Apollo.io enrichment stub — no network calls. When implemented, return verified
 * decision-maker email + confidence for CSV "Apollo Enrichment" column.
 */
export type ApolloEnrichmentResult = {
  direct_email: string | null;
  decision_maker_name: string | null;
  apollo_confidence: number | null;
};

export function enrichWithApollo(practiceName: string, website: string, city: string): ApolloEnrichmentResult {
  void practiceName;
  void website;
  void city;
  return { direct_email: null, decision_maker_name: null, apollo_confidence: null };
}

File: enrichment/apollo_stub.py
# APOLLO INTEGRATION STUB — implement when Apollo API key is available.
# See https://apolloio.github.io/apollo-api-docs/ for enrichment endpoint.
# Expected cost: ~$0.05-0.10 per record at standard tier.
# Wire this into the enrichment stage in pipeline.py after the email validator step.


def enrich_with_apollo(practice_name: str, website: str, city: str) -> dict:
    """
    When implemented, returns e.g.:
    {"direct_email": str | None, "decision_maker_name": str | None, "apollo_confidence": float | None}
    """
    return {"direct_email": None, "decision_maker_name": None, "apollo_confidence": None}
